### PR TITLE
Help contributors wrap code with a codeblock in the bug reporte template

### PR DESCRIPTION
**Pull request by harupy**
Signed-off-by: harupy <hkawamura0130@gmail.com>

## Related Issues/PRs

 #xxx

## What changes are proposed in this pull request?

This PR is an attempt to encourage bug reporters to wrap code in a code block and **reduce stress when triaging issues**.

https://user-images.githubusercontent.com/17039389/209124231-f7d5c917-f098-4493-8ad0-ca1b3396b197.mov

## How is this patch tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [x] Manual tests (describe details, including test results, below)

Tested in my fork: https://github.com/harupy/mlflow/issues/new?assignees=&labels=bug&template=bug_report_template.yaml&title=%5BBUG%5D

## Does this PR change the documentation?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly in the documentation preview.

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report_template.yaml [hunk 1]
```diff
@@ -72,26 +72,33 @@ body:
         mlflow.doctor()
         ```
         Then, make sure the printed out information matches what you expect and paste it (with sensitive information masked) in the box below. If you know the command that was used to launch your tracking server (e.g. `mlflow server -h 0.0.0.0 -p 5000`), please provide it.
+      value: |
+        ```
+        < REPLACE ME (DO NOT REMOVE BACKTICKS) >
```
Comment (harupy) [inline .github/ISSUE_TEMPLATE/bug_report_template.yaml:77]: Let me know if there is a better message :)

File: .github/ISSUE_TEMPLATE/bug_report_template.yaml [hunk 2]
```diff
@@ -72,26 +72,34 @@ body:
         mlflow.doctor()
         ```
         Then, make sure the printed out information matches what you expect and paste it (with sensitive information masked) in the box below. If you know the command that was used to launch your tracking server (e.g. `mlflow server -h 0.0.0.0 -p 5000`), please provide it.
+      value: |
+        <!-- PLEASE KEEP BACKTICKS AND CHECK PREVIEW -->
+        ```
```
Comment (BenWilson2) [inline .github/ISSUE_TEMPLATE/bug_report_template.yaml:77]:
```suggestion
        ``` shell
```

File: .github/ISSUE_TEMPLATE/bug_report_template.yaml [hunk 3]
```diff
@@ -101,40 +109,76 @@ body:
         with mlflow.start_run():
             mlflow.sklearn.log_model(model, "model")
         ```
+
+      value: |
+        <!-- PLEASE KEEP BACKTICKS AND CHECK PREVIEW -->
+        ```
```
Comment (BenWilson2) [inline .github/ISSUE_TEMPLATE/bug_report_template.yaml:115]:
```suggestion
        ``` python
```

Comment (BenWilson2) [reply .github/ISSUE_TEMPLATE/bug_report_template.yaml:115]: The color coding helps read walls of script :)

File: .github/ISSUE_TEMPLATE/bug_report_template.yaml (hunk 1 above)
Comment (BenWilson2) [reply .github/ISSUE_TEMPLATE/bug_report_template.yaml]: I like it.

**Review by BenWilson2 — APPROVED** [review]
LGTM! Up to you if you want to add the formatting hints to the pre-populated code blocks

File: .github/ISSUE_TEMPLATE/bug_report_template.yaml (hunk 3 above)
Comment (harupy) [reply .github/ISSUE_TEMPLATE/bug_report_template.yaml:115]: The code could be R or Java :)

**Review by WeichenXu123 — APPROVED** [review]
